app/assets/javascripts/discourse/app/components/composer-editor.hbs

```diff
@@ -43,6 +43,7 @@
     @topicId={{this.composer.model.topic.id}}
     @categoryId={{this.composer.model.category.id}}
     @onSetup={{this.setupEditor}}
+    @isUploading={{this.composer.disableSubmit}}
```

Comment:
super nit: maybe being closer to the original `disableSubmit` will be clearer

Reply:
Yah fair enough :+1: